doc/pipeline_process.md

```diff
@@ -13,31 +13,61 @@ The generally accepted recommendations for its use are described below:
+- Alternatively, you can open a `draft`.
 
+### When is a PR considered ready to move forward?
 
-### When is a PR considered to be Ready for testing by QA team?   
-Ready for testing PR should meet the following criteria: 
+Ready for testing, a PR should meet the following criteria:
 
-1. Reviewed and has at least 1 approval
-2. Rebased to `develop` branch (both `status-mobile` and `status-go` if needed, depending on what part has changes) 
-3. All possible conflicts have been resolved 
-4. Has the label: `request-manual-qa`
+1. Reviewed and has at least 1 approval.
+2. Rebased to the `develop` branch (both `status-mobile` and `status-go` if needed, depending on what part has changes).
+   - **NOTE:** QAs can use the `Update with rebase` button from GitHub in case if:
+        - The PR has no `status-go` changes.
+        - The PR has no conflict with the develop branch.
+3. All possible conflicts have been resolved.
+4. Has the label: `request-manual-qa` or `skip-manual-qa`.
+5. PRs **MUST** identify what area is affected and should have a description.
 
+**NOTE:** Make sure that QAs are OK with that
+
+### Adding `skip-manual-qa`
+
+- Please ask another team member before adding the `skip-manual-qa` label (or here in chat) so that there's a second opinion.
+- The PR should have a proper reasoning why manual QA is skipped.
+- The PR should include the steps of testing that has been done by the developer prior to moving it forward.
 **From the perspective of a developer it means that once work on PR is finished:**
 
 1. It should be rebased to the latest `develop`. If there are conflicts - they should be resolved if possible.
 2. If the PR was in the `Contributor` column - it should be moved to `Review` column.
 3. Wait for the review.
 4. Make sure that after review and before requesting manual QA your PR is rebased to current develop.
-5. Once the PR has been approved by reviewer(s) - label `request-manual-qa` should be applied to the PR
-6. Move PR to the E2E column when it is ready for testing (**mandatory for all PRs**). That will also trigger e2e tests run. QAs are monitoring PRs from E2E column and take it into test. 
+6. **MUST:** Move PR to the E2E column when it is ready for testing (**mandatory for all PRs**). That will also trigger e2e tests run. QAs are monitoring PRs from E2E column and take it into test. 
 
 After that - PR will be taken into manual testing by the QA team.
 
+### E2E tests and analyzing the results
+
+This step cannot be skipped. So, at least one comment from the `status-im-auto` bot with results is a prerequisite for moving forward.
+Information on how to analyze tests can be found [here](https://github.com/status-im/status-mobile/blob/develop/doc/how-to-launch-e2e.md).
+Tests might be flaky, as they depend on infrastructure - SauceLabs and Waku.
+
+If there are `Failed tests` and you are not sure about the reason, you can always ping the mobile QAs for help (preferably in PRs).
```

Comment:
I think you also can mention that devs cab use our group tag `@status-im/mobile-qa`